src/styles/profile.css

```diff
@@ -6,18 +6,38 @@
 
 .reserved-rocket {
   border: 1px solid lightgrey;
-  padding: 3% 3%;
   margin: 0;
+  padding: 4%;
+  font-size: 20px;
```

Comment:
Also decrease the padding.
padding : 2% 4%;

Reply:
done